app/views/pages/search_item.js.erb

```diff
@@ -5,14 +5,32 @@ $("<%= escape_javascript(render 'pages/search_result', items: items) %>")
 
 $("#loading-layer") && $("#loading-layer").remove();
 
-$('select[name="item[category_id]"]').on('change', (e) => {
-  const classNames = {};
+
+{
+  var itemsResult = document.getElementById('search-items-result');
+  var itemNames = itemsResult.querySelectorAll('input[name="item[name]"]');
+
+  for(var i = 0; i < itemNames.length; i++) {
+    itemNames[i].addEventListener('keyup', function (ev) {
```

Comment:
Should we add logic for removing event listeners here? Same feedback for `itemsCategory`

Reply:
I'll create a technical debt for it, because I think it is trickier than it seems...